react-interactive/src/ConfigurableTable.tsx

```diff
@@ -65,12 +65,12 @@ export default function ConfigurableTable<T>(props: IProps<T>) {
     const [guid, setGuid] = React.useState<string>(CreateGuid());
     
     React.useEffect(() => {
-        if (props.cols.length !== colEnabled.length)
+        // We need to redo this set collumn here to capture function changes within collumns
+        setCollumns(props.cols.filter((c, i) => colEnabled[i]));
```

Comment:
This will fail if you add or remove a collumn. because if `props.col.length > colEnabled.length` it throws an `indexOutOfRange` exception

Reply:
Conditional added to ensure this doesn't happen